modules/zenith-public/lua/1-enum/zxi/mobHelpers.lua

```diff
@@ -22,4 +22,51 @@ zxi.mobHelpers.getRespawnVarName = function(mob, mobId)
     return fmt(template, mob:getName(), mob:getID())
 end
 
+zxi.mobHelpers.zoneInitReposition = function(zone, mobList)
+    if not zone or not mobList then
+        return
+    end
+
+    for mobName, cfg in pairs(mobList) do
+        -- query all spawns of this mob in the zone
+        local entities = zone:queryEntitiesByName(mobName)
+
+        if entities and #entities > 0 then
```

Comment:
I believe we can trust that entities is always a table

```suggestion
        if #entities > 0 then
```